javalib/src/main/scala/java/util/logging/ConsoleHandler.scala

```diff
@@ -0,0 +1,19 @@
+package java.util.logging
+
+// Unfortunately the spec require the use of System.err
+class ConsoleHandler extends StreamHandler(System.err, new SimpleFormatter) {
```

Comment:
The `Formatter` should be loaded from the `LogManager` and default to `new SimpleFormatter` if it is not defined there. The doc says: 


**Configuration**: By default each ConsoleHandler is initialized using the following LogManager configuration properties. If properties are not defined (or have invalid values) then the specified default values are used.
...
java.util.logging.ConsoleHandler.formatter specifies the name of a Formatter class to use (defaults to java.util.logging.SimpleFormatter).
...